### PR TITLE
Adding script to copy label file into derivatives/labels/sub-XXX/anat

### PR DESCRIPTION
**Pull request by lrouhier**
This PR aims at creating a script that copies files from sub-xxx/anat to derivatives/labels/sub-xxx/anat. 
For now, the script is called as follow:
move_file_SG -p path/to/results/file -s suffix_of_file
 
the script: 
* go to path/results/ 
* List all object in folder 
* Check if ./sub-xxx/anat/sub-xxx_SUFFIX.nii.gz exist
* Create directory derivatives/labels/sub-xxx/anat
* copy  sub-xxx/anat/sub-xxx_SUFFIX.nii.gz to  derivatives/labels/sub-xxx/anat
* return the number of moved file (just to check if there is no error) 

pros and cons of asking for fully file suffix: 
* cons:
You need to call it once for each contrast

* pros:
you don't have to test if all contrasts are present.

### CODE REVIEW

**Comment by jcohenadad** [comment]
@lrouhier ready for review?

**Comment by lrouhier** [comment]
Just a question I thought about this morning: In the results how is the structure? 
is it sub-xxx/anat/files ?

**Comment by jcohenadad** [comment]
> Just a question I thought about this morning: In the results how is the structure?
> is it sub-xxx/anat/files ?

yup! and for dwi: sub-xxx/dwi/files

**Comment by lrouhier** [comment]
Alright I'll do something more modulable for dwi then !

**Comment by jcohenadad** [comment]
ah! no wait, i think we only need vertebral labels, right? these are only present in anat

**Comment by jcohenadad** [comment]
to be more precise: afair, for Paul's project specifically, what we need is: sub-XXX_T1w_RPI_r_seg_labeled.nii.gz

**Comment by lrouhier** [comment]
So for now the usage would be ./move_files_derivatives.sh SUFFIX of file so for Paul's project 
`./move_files_derivatives.sh T1w_RPI_r_seg_labeled`
Should we keep it that way or hardcode the file suffix?

**Comment by jcohenadad** [comment]
> So for now the usage would be ./move_files_derivatives.sh SUFFIX of file so for Paul's project
> `./move_files_derivatives.sh T1w_RPI_r_seg_labeled`
> Should we keep it that way or hardcode the file suffix?

sounds good to me 👍 
however, could you pls make it a .py script?  i envision this folder to become a proper python package, with callable CLI scripts (and proper setup, etc.).

**Comment by lrouhier** [comment]
I get it. I can do it! will @ you when done.

**Comment by lrouhier** [comment]
@jcohenadad I think it is good. 

To call the CLI move_file_SG -p path/to/results -s suffix_of_file. 
I Updated the body of the PR with the explanation on how to use it. I don't know if the path argument is relevant  ?

File: processing/move_file_derivatives.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import os
```
Comment (jcohenadad) [inline processing/move_file_derivatives.py:1]: can you please add a header to briefly explain what this function does

Comment (lrouhier) [reply processing/move_file_derivatives.py:9]: Done in 48edbd7

**Comment by jcohenadad** [comment]
Note: currently the CLI call fails because utils cannot be imported. This will be fixed via #96

**Pull request merged**